waveorder/cli/settings.py

```diff
@@ -61,6 +61,7 @@ class FourierTransferFunctionSettings(MyBaseModel):
     yx_pixel_size: PositiveFloat = 6.5 / 20
     z_pixel_size: PositiveFloat = 2.0
     z_padding: NonNegativeInt = 0
+    z_focus_offset: Union[int, Literal["auto"]] = 0
```

Comment:
Should be specific to phase? [cross-link](https://github.com/mehta-lab/waveorder/pull/441/files#r2096566385)

Reply:
E.g. `PhaseTransferFunctionSettings`